dadi/frontend/components/DropdownItem/DropdownItem.jsx

```diff
@@ -2,17 +2,11 @@
 
 import { h, Component } from 'preact'
 
-import { Style } from 'lib/util'
+import Style from 'lib/Style'
 import styles from './DropdownItem.css'
 
 export default class DropdownItem extends Component {
   handleClick(event) {
-
-    // Don't stop propagation for anchor node
-    if (!this.props.href) {
-      event.stopPropagation()
```

Comment:
Yes, but you can still merge this PR. It was the presence of `stopPropagation()` (which I removed) that was breaking the router. In fact, I'm not sure we need it anymore, so all should be good.